podcasts/DateHeadingView.swift

```diff
@@ -25,7 +25,7 @@ class DateHeadingView: UIView {
 
         titleLabel = ThemeableLabel(frame: CGRect(x: 20, y: 10, width: bounds.width - 20, height: 30))
         titleLabel?.font = UIFont.systemFont(ofSize: 22, weight: UIFont.Weight.bold)
-        titleLabel?.textAlignment = .left
+        titleLabel?.textAlignment = .natural
```

Comment:
Someone should probably test this view using an RTL language in a simulator and confirm that this change doesn't conflict with other aspects of the layout of screens where this view is used.

In theory, this is indeed the right thing to use, so that it's adaptive to the LTR vs RTL layout. But if other parts of the screen layout—like manual frame computations, alignment of the date text with other parts of the screen…—were based on the supposition that this was always left-aligned for example, that could benefit from adjusting accordingly if needed.

Reply:
Just tested that and it's working just fine. :)